src/sagemaker_containers/training.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import importlib
+import subprocess
+
+import sys
+
+import sagemaker_containers.training_environment as env
+
+
+class Training(object):
+    def __init__(self, train_fn=None, module_name=None):
```

Comment:
` train_fn` and `module_name` are mutually exclusive. (As inferred from the existance of  `from_train_fn()` and `from_module()`. This class needs to throw an exception if it is manually constructed and passed both.

Reply:
@iquintero, I refactored the class to cover that scenario: https://github.com/mvsusp/sagemaker-container-support/pull/3/files#diff-35452821e8fa0a4ef79ea98492563a4aR35

Please, let me know your thoughts.